llvm/test/CodeGen/ARM/misched-branch-targets.mir

```diff
@@ -0,0 +1,169 @@
+# REQUIRES: asserts
```

Comment:
It's customary to put REQUIRES after the RUN lines, but either should work.

Reply:
Fixed. Thanks